content/trial_getting_started/set_trial_manager/other-deployments.md

```diff
@@ -17,9 +17,33 @@ Read about the differences between our {{< param product_name >}} versions, and
 ## {{< param product_name >}} Premium (Premium download / Hosted service trial)
 
 
+### 5.2.0
+
+v5.2.0 is the latest __minor__ release. It can be used for new installations, upgrade from 5.0.5 or earlier releases, or as an update to a v5.1.x system.
```

Comment:
As far as I know, v5.2.0 cannot be used as an update to v5.1.x. Only patch versions can be used for this.
(Given that we refer to "update" as running `cfy_manager upgrade <version>` and not as running snapshot restore).

Reply:
It can be used as an update- because we avoid DB updates during patch releases it is expected to be safe to upgrade in that way (though I still prefer the old upgrade process).

Reply:
Following our discussion, this upgrade process is not tested. [This JIRA](https://cloudifysource.atlassian.net/browse/RD-1986) describes it.